.pipelines/cd.yml

```diff
@@ -0,0 +1,59 @@
+pool:
+  vmImage: ubuntu-18.04
+
+trigger: none
+resources:
+  pipelines:
+  - pipeline: IoTStarterCI
+    project: IoTStarter

```

Comment:
>IoTStarter [](http://example.com/codeflow?start=13&length=10)

should the IoTStarter be parametrized in case user decides to name the project something else?